checkov/github/dal.py

```diff
@@ -107,7 +109,9 @@ def get_repository_webhooks(self) -> dict[str, Any] | None:
     def persist_repository_webhooks(self) -> None:
         repository_webhooks = self.get_repository_webhooks()
         if repository_webhooks:
-            BaseVCSDAL.persist(path=self.github_conf_file_paths["repository_webhooks"], conf=repository_webhooks)
+            for idx, item in enumerate(repository_webhooks):
+                path = str(self.github_conf_file_paths["repository_webhooks"])
+                BaseVCSDAL.persist(path=path.replace(".json", str(idx) + ".json"), conf=[item])    # type: ignore
```

Comment:
can you add some logs here for `path=path.replace(".json", str(idx) + ".json")` ?

Reply:
We have the logs inside the persist method, so we'll know what we're writing and to where